test/functional/services/browser.js

```diff
@@ -18,11 +18,20 @@
  */
 
 import { modifyUrl } from '../../../src/core/utils';
+import Keys from 'leadfoot/keys';
 
 export function BrowserProvider({ getService }) {
   const leadfoot = getService('__leadfoot__');
 
   return new class BrowserService {
+
+    /**
+     * Gets Keys to call keyboard events
+     */
+    getKeys() {
```

Comment:
I'd personally prefer if this was just `keys = Keys`, so we don't have to call `getKeys()` when using it and instead can do something like `await input.pressKeys(browser.keys.NULL); // Release modifier keys`